.github/actions/build_clio/action.yml

```diff
@@ -4,12 +4,18 @@ inputs:
   target:
     description: Build target name
     default: all
+  substract_threads:
```

Comment:
This is not strictly needed anymore (was a test to see if `heavy` would survive) but we can keep it in case we need it in the future.